packages/data-grid/src/quickFilter/QuickFilter.md

```diff
@@ -49,6 +49,10 @@ const filters = [
       },
     ],
   },
+  {
```

Comment:
I get this component doesn't currently have any written docs and im not asking you to completely write a whole page of content. But I think we need to show this example separate from the rest and explain this is only a disabled state. And that a null value is an acceptable one value and should be treated differently. Would even be worth showing the null value example from comply.